test/aievec/mul_mul_f32.mlir

```diff
@@ -1,13 +1,12 @@
 // RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=8" --aie-vectorize="shift=10" -split-input-file | FileCheck %s
-// XFAIL: *
 
-func @mul_mul (%A: memref<2048xf32>, %B: memref<2048xf32>, %C: memref<2048xf32>, %cc: f32) {
+func.func @mul_mul (%A: memref<2048xf32>, %B: memref<2048xf32>, %C: memref<2048xf32>, %cc: f32) {
 // CHECK-LABEL: func @mul_mul
 // CHECK:  %2 = aievec.upd %arg0[%arg4] {index = 0 : i8, offset = 0 : i32} : memref<2048xf32>, vector<8xf32>
 // CHECK:  %3 = aievec.upd %arg1[%arg4] {index = 0 : i8, offset = 0 : i32} : memref<2048xf32>, vector<8xf32>
-// CHECK:  %4 = aievec.mul %1, %2 {xoffsets = "0x76543210", xstart = "0", zoffsets = "0x76543210", zstart = "0"} : vector<16xf32>, vector<8xf32>, !aievec.acc<8xf32>
-// CHECK:  %5 = aievec.srs %4 {shift = 10 : i8} : !aievec.acc<8xf32>, vector<8xf32>
-// CHECK:  %6 = arith.mulf %5, %3 : vector<8xf32>
+// CHECK:  %4 = aievec.mul %1, %2 {xoffsets = "0x76543210", xstart = "0", zoffsets = "0x76543210", zstart = "0"} : vector<16xf32>, vector<8xf32>, vector<8xf32>
```

Comment:
where is %1 defined?

Reply:
Well somewhere but the lit test isn't checking it.